doc/CMakeLists.txt

```diff
@@ -72,4 +72,4 @@ if (BUILD_SPHINX_DOCUMENTATION)
     else (SPHINX_FOUND)
         message(FATAL_ERROR "Sphinx need to be installed to generate the doxygen documentation")
     endif (SPHINX_FOUND)
-endif(BUILD_SPHINX_DOCUMENTATION)
\ No newline at end of file
+endif(T8CODE_BUILD_SPHINX_DOCUMENTATION)
```

Comment:
```suggestion
endif( T8CODE_BUILD_DOCUMENTATION_SPHINX )
```

Reply:
My mistake, fixed in in the following commit.